File: TecBankAPI/Models/Cliente.cs
namespace TecBankAPI.Models;

public class Cliente
{
    
    public string? Id { get; set; }
    public string? NombreCompleto { get; set; }
    public string? Cedula { get; set; }
    public string? Direccion { get; set; }
    public string? Telefono { get; set; }
    public decimal IngresoMensual { get; set; }
    public string? TipoCliente { get; set; } // "Físico" o "Jurídico"
    public string? Usuario { get; set; }
    public string? Password { get; set; }
}

File: src/mobile/tecbank-cli/app/src/main/java/com/example/tecbankapp/models/Cliente.kt

package com.example.tecbankapp.models

data class Cliente(
    val id: String? = null,
    val nombreCompleto: String,
    val cedula: String,
    val direccion: String,
    val telefono: String,
    val ingresoMensual: Double,
    val tipoCliente: Int,
    val usuario: String,
    val password: String
)
